client/my-sites/importer/newsletter/subscribers/paid-subscribers/map-plan.tsx

```diff
@@ -64,11 +66,7 @@ export function MapPlan( {
 	tierToAdd,
 	selectedProductId,
 }: MapPlanProps ) {
-	let active_subscriptions = '';
```

Comment:
Simplified by not storing string in a variable and moved inline into the component.